lib/src/test/scala/TestHelper.scala

```diff
@@ -22,6 +22,7 @@ object TestHelper extends Matchers {
   lazy val apidocApiService: Service = parseFile(s"/examples/apidoc-api.json")
   lazy val dateTimeService: Service = parseFile(s"/examples/date-time-types.json")
   lazy val builtInTypesService: Service = parseFile(s"/examples/built-in-types.json")
+  lazy val scalaKeywordsService: Service = parseFile(s"/examples/response-with-reserved-scala-keyword.json")
```

Comment:
Is there something missing here? A test that validates the keywords are escaped?

Reply:
This is a sample file referenced in Play26ControllersSpec.  AFAICT the existing examples don't have scala keywords in them.